scripts/postinstall.js

```diff
@@ -30,8 +30,48 @@ const removeUnwantedFolders = async (root, unwantedNames) => {
 
   return promises;
 };
+
+const patchFile = async (file, patch) => {
+  console.log(`Patching ${file}`);
```

Comment:
Do we need debug log here?

Reply:
I think we should. Did you have another thought?

Reply:
I was just checking if it is necessary. I am good with it.